src/server/bootstrap/helmet.ts

```diff
@@ -0,0 +1,21 @@
+import helmet from 'helmet'
+import config from '../config'
+import parseDomain from '../utils/domain'
+
+const cspOnlyReportViolations = config.get('cspOnlyReportViolations')
+const cspReportUri = config.get('cspReportUri')
+const sentryDns = config.get('sentryDns')
+
+export default helmet({
+  contentSecurityPolicy: {
+    directives: {
+      defaultSrc: ["'self'"],
+      styleSrc: ["'self'", "'unsafe-inline'", 'https://fonts.googleapis.com/'],
+      fontSrc: ["'self'", 'https://fonts.gstatic.com/'],
+      connectSrc: ["'self'", ...(sentryDns ? [parseDomain(sentryDns)] : [])],
+      ...(cspReportUri ? { reportUri: cspReportUri } : {}),
```

Comment:
You didn't put the report Uri in an array in the end?

Reply:
In Sentry's example for how to set up report-uri, they don't include report-uri in the connect-src directive
```
def middleware(request, response):
    response['Content-Security-Policy-Report-Only'] = \
        "default-src 'self'; " \
        "report-uri https://o545501.ingest.sentry.io/api/5667596/security/?sentry_key=5049a239ada645f5900b090068b008bf"
    return response
```

I also tested it with a personal sentry i set up. The CSP reports go through.
![image](https://user-images.githubusercontent.com/18087283/110414103-3708ba00-80ca-11eb-9639-ba361f36045a.png)

Reply:
In Sentry's example for how to set up report-uri, they don't include report-uri in the connect-src directive
```
def middleware(request, response):
    response['Content-Security-Policy-Report-Only'] = \
        "default-src 'self'; " \
        "report-uri https://o545501.ingest.sentry.io/api/5667596/security/?sentry_key=XXXXXXXXXXXXXXXXX"
    return response
```

I also tested it with a personal sentry i set up. The CSP reports go through.
![image](https://user-images.githubusercontent.com/18087283/110414103-3708ba00-80ca-11eb-9639-ba361f36045a.png)